### PR TITLE
small additions to the docs

### PR DESCRIPTION
**Pull request by tobiasd**

### CODE REVIEW

File: INSTALL.txt [hunk 1]
```diff
@@ -140,7 +140,17 @@ $a->config['system']['addon'] = 'js_upload,poormancron';
 
 and save your changes.
 
-9. (Optional) Reverse-proxying and HTTPS
+9. (Optional) Set up a backup plan
+
+Bad things will happen. Be them a hardware failure, a currupted database
```
Comment (annando) [inline INSTALL.txt]: "Let there be a ..." sounds better in my ears.

Comment (tobiasd) [inline INSTALL.txt]: Mmh... ok, I'll change it later

File: INSTALL.txt [hunk 2]
```diff
@@ -140,7 +140,17 @@ $a->config['system']['addon'] = 'js_upload,poormancron';
 
 and save your changes.
 
-9. (Optional) Reverse-proxying and HTTPS
+9. (Recommended) Set up a backup plan
+
+Bad things will happen. Let there be them a hardware failure, a currupted
```
Comment (annando) [inline INSTALL.txt]: "Let there be _them_ a hardware failure" - without "them", please ;)

File: INSTALL.txt [hunk 3]
```diff
@@ -140,7 +140,17 @@ $a->config['system']['addon'] = 'js_upload,poormancron';
 
 and save your changes.
 
-9. (Optional) Reverse-proxying and HTTPS
+9. (Recommended) Set up a backup plan
+
+Bad things will happen. Let there be a hardware failure, a currupted
```
Comment (annando) [inline INSTALL.txt]: Sorry, I just detected something else: "corrupted" would be better.

File: INSTALL.txt [hunk 4]
```diff
@@ -140,7 +140,17 @@ $a->config['system']['addon'] = 'js_upload,poormancron';
 
 and save your changes.
 
-9. (Optional) Reverse-proxying and HTTPS
+9. (Recommended) Set up a backup plan
+
+Bad things will happen. Let there be a hardware failure, a corupted
```
Comment (annando) [inline INSTALL.txt]: You really like that word ;-) "corrupted" would be even better ;-)

Comment (tobiasd) [inline INSTALL.txt]: just raising the commit numbers, the online editor also helps a lot there ;-)

**Pull request merged**